lib/tags/group-settings.tag

```diff
@@ -166,6 +166,7 @@
                                 self.group.title = title;
                                 self.refs.title.classList.remove('graphjs-error');
                                 self.refs.title.classList.add('graphjs-success');
+                                self.refs.title.setAttribute('data-savesuccesstext', i18n.saveSuccessText);
```

Comment:
Please remove this change. self.refs.title refers to an input element and input elements do not accept ::before or ::after pseudo-elements.

Reply:
Same as above